client/me/billing-history/style.scss

```diff
@@ -108,6 +108,11 @@
 }
 
 .billing-history__service-name {
+	small,
```

Comment:
Any difference if we apply this directly to `.billing-history__service-name `, rather than to nested `small` and `strong`?